bin/reth/src/config.rs

```diff
@@ -78,3 +80,12 @@ impl Default for SenderRecoveryConfig {
         Self { commit_threshold: 5_000, batch_size: 1000 }
     }
 }
+
+/// Configuration for each stage in the pipeline.
+#[derive(Debug, Clone, Default, Deserialize, Serialize)]
+pub struct DiscoveryConfig {
+    /// Preferred nodes to connect to.
+    pub trusted_nodes: Vec<String>,
```

Comment:
This should be NodeRecord, and let's impl any missing traits if needed for it

Reply:
yep, I guess we can add `Serialize` Deserialize Support to `NodeRecord` via FromStr + Display

can be a separate PR if you like @MegaRedHand

Reply:
perhaps we even move NodeRecord to primitives

Reply:
I'm going to make these changes in a separate PR then, as per your suggestion.

Reply:
Imo we should try to not end up copy/pasting the peers config (and network config) to this module as we would end up doing double work. Ideally the network crate would have a `serde` feature to deserialize/serialize the relevant configs. Because of that, I think this particular part should be moved to a separate section of the current config as `peers` will probably clash at some point